scripts/banking/importFilteredOperations.js

```diff
@@ -37,6 +45,13 @@ module.exports = {
       })
       .map(e => ({ ...e, _type: DOCTYPE_OPERATIONS }))
       .slice(0, nOperations)
+      .sort(() => {
+        if (typeof deterministic === 'string') {
+          return randomGenerator() - 0.5
+        } else {
+          return deterministic ? 0 : Math.random() - 0.5
```

Comment:
I don't quite understand this `- 0.5`?

Reply:
Sort expects a number `x` and inverts elements `a` and `b` if `x < 0`. `Math.random()` gives a number between 0 and 1 so it has a probability 1/2 of inverting two elements

Reply:
Ok, but why would you want to change the sort order for the deterministic case? If 2 executions give different result, it does not sound very deterministic :) Moreover, If you have a deterministic string , the change order is 50/50, but it's always the same if it's boolean